content/news/2022-06-10-mierune-announcement.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "MIERUNE Inc. supports MapLibre with USD 10'000"
+date: "2022-06-10"
+categories: ["announcements"]
+authors: [wipfli]
+draft: false
+---
+
+Recently, [MIERUNE Inc.](https://www.mierune.co.jp/?lang=en) from Japan decided to support MapLibre financially with a generous donation of USD 10'000, see https://opencollective.com/maplibre. This is the largest ammount that MapLibre has received to date.
+
+Thank you MIERUNE Inc., we are very proud that you trust MapLibre and support us!
+
+To better understand MIERUNE's background and motivation, we asked them a few questions which they answered below:
+
+## What is MIERUNE Inc. and how many people work there?
+
+We are a Japanese software company specializing in geographic information technologies. We provide solutions for a variety of fields, including communications, logistics, transportation, medicine, weather, and the public sector. We are based in Hokkaido, the second largest island in the north part of Japan. We currently have 16 members from a variety of backgrounds; web, geography, construction, and civil services, to name a few.
+
+## Why do you support MapLibre?
+
+We strongly believe in the open-source culture. Continuous support for the OSS is essential to make innovations happen. We think that contributing to the OSS ecosystems is an important activity as a company benefiting from them.
+
+We would like to support MapLibre's growth in terms of providing a variety of solution options and opportunities to the community.
+
+We see a growing number of MapLibre users here in Japan, and we expect them to increase even faster.
+
+We want to use this contribution as a stepping stone to participate in projects around the world that take full advantage of MapLibre's capabilities, and feedback on the needs of the users to the development team, as well as directly contribute to the development itself (we already have contributed to the codebase before sponsoring the project).
+
+## Have you supported other open-source projects in the past?
+
+- QGIS.org - Sustaining Member: https://qgis.org/en/site/about/sustaining_members.html
+- Funded "Vector Tile native support in QGIS" Crowdfunding Project: https://www.lutraconsulting.co.uk/crowdfunding/vectortile-qgis/
+- OSGeoJP (Japan Chapter of OSGeo Foundation) - Institutional Sponsor: https://www.osgeo.jp/sponsor
+- OpenStreetMap Foundation Japan - "State of the Map" Conference Sponsor, 2017-2020): https://stateofthemap.jp/ (As we do not have a conference yet after 2020)
```

Comment:
```suggestion
[OpenStreetMap Foundation Japan - "State of the Map" Conference Sponsor, 2017-2020)](https://stateofthemap.jp/) (As we haven't had a conference since 2020 due to the global situation)
```